src/server/console/logind_console_services.cpp

```diff
@@ -934,10 +937,12 @@ mir::LogindConsoleServices::~LogindConsoleServices()
 
 void mir::LogindConsoleServices::on_lock()
 {
+    mir::log_info("Setting the locked hint for the session to true");
```

Comment:
Also a good `log_info`, although:
```suggestion
    mir::log_info("Notifying logind of session lock");
```